src/main/java/uk/ac/ebi/subs/fileupload/eventhandlers/FileHelper.java

```diff
@@ -15,6 +15,7 @@ public static File convertTUSFileInfoToFile(TUSFileInfo tusFileInfo) {
         file.setFilename(tusFileInfo.getMetadata().getFilename());
         file.setTotalSize(tusFileInfo.getSize());
         file.setUploadedSize(tusFileInfo.getOffsetValue());
+        file.setTargetPath("/path/to/the/file/" + tusFileInfo.getMetadata().getFilename());
```

Comment:
This looks a bit odd, should it be here?

Reply:
No, it should not. Thanks
I will remove it